assets/js/Line.jsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Sign from './Sign';
+
+function lineSigns(line) {
+  if (line === '') {
```

Comment:
This feels more like an `if` / `else if` chain to me than a series of `if`s. I know with `return` in each one it's effectively the same, but I think they convey a different intent. Also, rather than `'Mattapan'` being the fallback, how about making it `line === 'Mattapan'` and making this empty array be the final `else`?

Reply:
the linter told me not to use else after a return, but i can move that to the end